schemas/2018-12-01/Microsoft.Network.json

```diff
@@ -89,6 +89,67 @@
       ],
       "description": "Microsoft.Network/applicationGateways"
     },
+    "ApplicationGatewayWebApplicationFirewallPolicies": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies"
```

Comment:
>ApplicationGatewayWebApplicationFirewallPolicies [](start = 31, length = 48)

add test?

Reply:
Added some tests.
It would be great if related specs developer could review these tests and suggest improvements or add more there if required. @azhao155, could you please take a look?